gun/build.boot

```diff
@@ -0,0 +1,28 @@
+(set-env!
+ :resource-paths #{"resources"}
+ :dependencies '[[cljsjs/boot-cljsjs "0.5.2"  :scope "test"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "0.3.4")
+(def +version+ (str +lib-version+ "-0"))
+
+(task-options!
+ pom  {:project     'cljsjs/gun
+       :version     +version+
+       :description "GUN is a realtime, distributed, offline-first, graph database engine"
+       :url         "https://github.com/amark/gun"
+       :scm         {:url "https://github.com/cljsjs/packages"}
+       :license     {"MIT" "https://opensource.org/licenses/MIT"}})
+
+(deftask package []
+  (comp
+   (download :url (format "https://github.com/amark/gun/archive/v%s.zip" +lib-version+)
+             :checksum "70EE5F8EA4C62B090E4896A96DA40A71"
+             :unzip true)
+   (sift :move {#".*gun\.js$"   "cljsjs/gun/production/gun.inc.js"})
```

Comment:
This package does not come with a minified version yet. Judging from their github. This should come with the next release